chat-bots/src/Control/Monad/ListT.hs

```diff
@@ -107,7 +143,7 @@ instance Bifunctor ListF where
 
 -- | The empty 'ListT'.
 emptyListT :: Applicative m => ListT m a
-emptyListT = ListT $ pure NilF
+emptyListT = absurd <$> Functor.introduce ()
```

Comment:
Since you've already gone to the trouble of defining an `Alternative` instance for `ListT`s, you can just use `emptyListT = empty` here (or you could just remove this like you did for `alignListT`).

Reply:
The trouble here is the `Alternative` instance has a `Monad` constraint which is only needed for the `combine` used in `<|>`.